cea/optimization/slave/cooling_resource_activation.py

```diff
@@ -215,64 +198,76 @@ def cooling_resource_activator(Q_thermal_req,
 
             E_BaseVCC_WS_req_W += E_pump_WS_req_W
 
-
-        elif T_source_average_Lake_K <= (T_district_cooling_supply_K - DT_COOL):  # bypass, do not use chiller but use heat exchange to cool the water directly
+        # if the water body temperature is lower than the temperature required by the district cooling network's heat
+        # exchanger, bypass the vapour compression chiller and use the water body for free cooling
+        elif T_source_average_water_body_K <= (T_district_cooling_supply_K - DT_COOL):
             E_pump_WS_req_W = calc_water_body_uptake_pumping(Q_BaseVCC_WS_gen_W,
                                                              T_district_cooling_return_K,
                                                              T_district_cooling_supply_K)
             E_BaseVCC_WS_req_W = E_pump_WS_req_W
         else:
-            print("no lake water source baseload VCC was used")
-        Q_therm_Lake_W -= Q_BaseVCC_WS_gen_W  # discount availability
-        Q_cooling_unmet_W = Q_cooling_unmet_W - Q_BaseVCC_WS_gen_W - Qc_from_storage_W + Qc_BaseVCC_WS_gen_storage_W # the provided cooling equals the produced cooling plus the cooling from storage minus the stored cooling
+            print("no lake water source base load VCC was used")
```

Comment:
Should this be "water body" instead of "lake water"?

Reply:
Agreed. Changed in the last commit.